client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/entity/CarColor.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.domain.entity
+
+enum class CarColor(val hexadecimal: Long) {
+    RED(0xffF47373),
+    YELLOW(0xffF8EC7E),
+    GREEN(0xff80E5AB),
+    BLUE(0xff77DEEE),
+    WHITE(0xffFFFFFF),
+    GREY(0xffAFAFAF),
+    BLACK(0xff3F3F3F)
+}
```

Comment:
i've already mentioned the entity would not know the value of the car color

Reply:
It should know it here, it's just an **integer** to represent **our** colors in the systems, it isn't a compose Color class.